cloudinit/net/activators.py

```diff
@@ -184,6 +184,30 @@ def bring_down_interface(device_name: str) -> bool:
         cmd = ["nmcli", "device", "disconnect", device_name]
         return _alter_interface(cmd, device_name)
 
+    @classmethod
+    def bring_up_interfaces(cls, device_names: Iterable[str]) -> bool:
+        """Activate network
+
+        Return True on success
+        """
+        state = subp.subp(
+            [
+                "systemctl",
+                "show",
+                "--property=SubState",
+                "NetworkManager.service",
+            ]
+        ).stdout
+        if "SubState=running" != state:
+            LOG.warning(
+                "Expected NetworkManager SubState=running, but detected: %s",
+                state,
+            )
+        return _alter_interface(
```

Comment:
was there supposed to be an `else` here? The final service code only does the reload or restart if network manager is already started.....though if it isn't running already we have much bigger problems.

Reply:
> was there supposed to be an `else` here? The final service code only does the reload or restart if network manager is already started.....though if it isn't running already we have much bigger problems.

The intent here was just to warn about the bigger problems in the case that it isn't already running.